test/close.test.js

```diff
@@ -23,6 +23,25 @@ test('close callback', t => {
   })
 })
 
+test('close promise', t => {
+  t.plan(4)
+  const fastify = Fastify()
+  fastify.addHook('onClose', onClose)
+  function onClose (instance) {
```

Comment:
this is expected. The instance parameter is now gone, and it’s just a callback. Just remove it and it should work.

Still, is probably something we should fix in avvio.

Reply:
I think needs a patch on this line (?)

https://github.com/mcollina/avvio/blob/2089008d28ca5d81209e2ce374b4ab927eb66e3d/boot.js#L221

to default to `callback` if parameter is not a function or just throw maybe.

Reply:
That’s needed, but not what is happening here.

This is a breaking change between avvio@5 and avvio@6, and 100% expected.